### ISSUE TITLE
How to use data in csv file row wise to be used per request via newman?

### ISSUE THREAD

**Issue by Sjain01** [postmanlabs/newman#2120]
I have bunch of requests in my postman collection for example :- Request 1 Request 2 ... Request N

For each of these requests , I want to pass a client id which is unique per request. I have created a data file with those client ids. So the data in CSV file is as follows : - Client Id 1 2 .. N

My requirement is to use Client ID 1 in Request 1 , Client ID 2 in Request 2 instead of iterating Client ID 1 though the entire collection.

So basically data in CSV file to be used row wise in all the requests.

Is there any such feature or a workaround which can help achieving above functionality ?

**Comment by codenirvana** [comment]
This is not possible as it will break the fundamental workflow of the data files.
I suggest you rely upon the data file's rows and assign a programmatic counter to the variable name.
